packages/language-support/src/lexerSymbols.ts

```diff
@@ -406,3 +406,6 @@ export const tokenNames: string[] = Array.from({ length }).map(
 );
 
 export const keywordNames = new Set(lexerKeywords.map((i) => tokenNames[i]));
+export const operatorSymbols = new Set(
+  lexerOperators.map((i) => literalNames[i].replaceAll("'", '')),
```

Comment:
The lexer operators are wrapped up in `'` inside ANTLR, e.g. the vocabulary has `'*'` instead of `*`, so we neeed to clean that manually here

Reply:
Are there any literals containing the `'` sign naturally where this is problematic?

Reply:
Not for the lexer operators